### PR TITLE
Add new entries to gazetteers

### PR DESCRIPTION
**Pull request by dasulit**
Add a handful of new gazetteer entries that reflect features highlighted in style components.

**Updated gazetteers:**
* worldwide/natural-labels.json
* worldwide/points-of-interest.json
* worldwide/road-network.json
* worldwide/transit.json

**New gazetteers:**
* worldwide/admin-boundaries.json
* worldwide/buildings.json
* worldwide/land-and-water.json
* worldwide/terrain.json
* worldwide/walking-cycling.json

@ClareTrainor for review.

### CODE REVIEW

File: worldwide/road-network.json [hunk 1]
```diff
@@ -234,6 +234,145 @@
...
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-122.667, 45.53]
+      },
+      "properties": {
+        "place_name": "Rose Quarter, Portland",
```
Comment (ClareTrainor) [inline worldwide/road-network.json]: Worth calling Portland, Washington so users don't get confused with Portland, Oregon?

File: worldwide/road-network.json [hunk 2]
```diff
@@ -234,6 +234,145 @@
...
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-74.067, 40.739]
+      },
+      "properties": {
+        "place_name": "Pulaski Skyway, Croxton",
```
Comment (ClareTrainor) [inline worldwide/road-network.json]: "Pulaski Skyway, Croxton, New Jersey"?  I didn't know where this was 😬

File: worldwide/road-network.json [hunk 3]
```diff
@@ -234,6 +234,145 @@
...
+        "place_name": "SoMa, San Francisco",
+        "zoom": 15,
+        "tags": ["road:service"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-77.044, 38.902]
```
Comment (ClareTrainor) [inline worldwide/road-network.json]: This coordinate location is pointing to a building. Could we update it to know if it is the road above or to the right?
Screen Shot 2019-12-13 at 2 21 33 PM

File: worldwide/terrain.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  "name": "terrain",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-77.034, -12.17]
+      },
+      "properties": {
+        "place_name": "Morro Solar, Chorrillos",
```
Comment (ClareTrainor) [inline worldwide/terrain.json]: Morro Solar, Chorrillos, Peru?

File: worldwide/walking-cycling.json [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-2.814, 56.349]
+      },
+      "properties": {
+        "place_name": "Old Course at St Andrews",
```
Comment (ClareTrainor) [inline worldwide/walking-cycling.json]: Nice choice for the golf course ⛳️

File: worldwide/walking-cycling.json [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        "zoom": 14,
+        "place_description": "Pistes, trails, and tracks.",
+        "tags": ["road:piste", "road:trail", "road:track"]
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [2.3, 48.858]
```
Comment (ClareTrainor) [inline worldwide/walking-cycling.json]: Possibly move this point to the intersection.
Screen Shot 2019-12-13 at 2 30 19 PM

**Comment by dasulit** [comment]
Thanks @ClareTrainor for looking closely! Updated based on your suggestions. Gonna merge!

**Pull request merged**